doc/110-Installation/30-API-Check-Forwarder.md

```diff
@@ -106,3 +106,70 @@ Enable-IcingaFrameworkApiChecks;
 ## EventLog Errors
 
 In case a check could not be executed by using this experimental feature, either because of timeouts or other issues, they are added with `EventId 1553` inside the EventLog for `Icinga for Windows`. A description on why the check could not be executed is added within the event output.
+
+## Icinga Communication to API
+
+With Icinga 2.14.0 and later, you can enable the Icinga Agent to natively communicate with the Icinga for Windows API, allowing checks being executed without having to start a PowerShell.
+
+This is a huge performance boost and should be **mandatory** on all Windows machines.
```

Comment:
@LordHepipud Now as I'm reading it... yes, all this is very nice, but could https://github.com/Icinga/icinga2/pull/8508/files also be a short-term solution for this?

Reply:
Isn't that something entirely different? Isn't the performance boost mentioned here about starting a PowerShell process and loading all the PowerShell Framework code for each check execution (or rather that with the API check, this is no longer needed)?